jobs/org_status.rb

```
@@ -1,11 +1,17 @@
 require 'rest_client'
 require 'xmlsimple'
 
-spreadsheet_root = "https://spreadsheets.google.com/feeds/cells/#{ENV['main_spreadsheet_key']}"
+SPREADSHEET_ROOT = "https://spreadsheets.google.com/feeds/cells/#{ENV['main_spreadsheet_key']}/od6/public/values"
+
+def fetch_with_retry(cell)
+  begin
+    value = XmlSimple.xml_in(RestClient.get("#{SPREADSHEET_ROOT}/#{cell}"))['content']['content']
+    sleep(2)
```

I'm still a bit wary of this because it could retry forever... Could we raise an error after the first couple of retries fail? It would be worth seeing what testing that scenario to see what the user experience is.